cypress/support/index.ts

```diff
@@ -251,8 +251,30 @@ Cypress.Commands.addAll({
         ]);
     },
     loadLangFile: () => {
+        cy.log('Load langs fixture');
```

Comment:
debug?

Reply:
its info message, if don't need it, I can drop it